web/html/src/components/FormulaForm.tsx

```diff
@@ -78,6 +82,8 @@ class FormulaForm extends React.Component<Props, State> {
       messages: [],
       warnings: [],
       errors: [],
+      sectionsExpanded: "collapsed",
```

Comment:
Here and elsewhere, I think it makes sense to replace this with an enum, unless any kind of a value can be stored here(?). A-la:

```ts
export enum SectionState {
  Expanded,
  Collapsed,
}

type Foo = {
  sectionsExpanded: SectionState;
};

const foo: Foo = {
  sectionsExpanded: SectionState.Expanded,
};
```